testutils/src/main/java/com/microsoft/identity/internal/testutils/labutils/KeyVaultAuthHelper.java

```diff
@@ -49,10 +50,28 @@
     private final static String MSSTS_CLIENT_ASSERTION_AUDIENCE = "https://login.microsoftonline.com/microsoft.com/oauth2/v2.0/token";
 
     private static KeyVaultAuthHelper sKeyVaultAuthHelper;
+    private final String mSecret;
+
+    private KeyVaultAuthHelper(final String secret) {
+        mSecret = secret;
+    }
 
     private KeyVaultAuthHelper() {
+        String secret = com.microsoft.identity.internal.testutils.BuildConfig.LAB_CLIENT_SECRET;
+        if (TextUtils.isEmpty(secret)) {
+            secret = getBuildConfigSecretFromClasspath()
+        }
```

Comment:
Can we just get from classpath directly? without first checking for it in test utils

Reply:
I want to give testUtils some sense of priority, to avoid disrupting anything that's currently in use.  If you're currently consuming testUtils as source, and aren't overriding the behavior by calling the methods that use the secret directly, then it should work just like it used to.

Reply:
And I can't find it in the class path anyway, without making build changes I'm not willing to make.